updates/3/updates.sql

```diff
@@ -0,0 +1,52 @@
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaC0104Cca91D167873B8601d2e71EB3D4D8c33e0", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaC0104Cca91D167873B8601d2e71EB3D4D8c33e0", "C", "Crowns", "CWS", 1612013846, NULL, "crowns", "CWS", "0xaC0104Cca91D167873B8601d2e71EB3D4D8c33e0");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xeEAA40B28A2d1b0B08f6f97bB1DD4B75316c6107", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xeEAA40B28A2d1b0B08f6f97bB1DD4B75316c6107", "C", "GOVI", "GOVI", 1608731186, NULL, "govi", "GOVI", "0xeEAA40B28A2d1b0B08f6f97bB1DD4B75316c6107");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x9F9c8ec3534c3cE16F928381372BfbFBFb9F4D24", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x9F9c8ec3534c3cE16F928381372BfbFBFb9F4D24", "C", "GraphLinq", "GLQ", 1615827809, NULL, "graphlinq-protocol", "GLQ", "0x9F9c8ec3534c3cE16F928381372BfbFBFb9F4D24");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x7825e833D495F3d1c28872415a4aee339D26AC88", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x7825e833D495F3d1c28872415a4aee339D26AC88", "C", "pTokens TLOS", "TLOS", 1613670383, NULL, "telos", "TLOS", "0x7825e833D495F3d1c28872415a4aee339D26AC88");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272", "C", "SushiBar", "xSUSHI", 1599311156, NULL, "xsushi", "", "0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x1eB754B8355d3185A7429109bde75e473434B26B", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x1eB754B8355d3185A7429109bde75e473434B26B", "C", "EASY V2", "EZ", 1619411513, NULL, "easyfi", "EZ", "0x1eB754B8355d3185A7429109bde75e473434B26B");
+*
+UPDATE assets SET swapped_for="_ceth_0x1eB754B8355d3185A7429109bde75e473434B26B" WHERE identifier="_ceth_0x913D8ADf7CE6986a8CbFee5A54725D9Eea4F0729";/*Migration of EeasyFi*/
```

Comment:
This seems like the wrong address.

`0x00AbA6fE5557De1a1d565658cBDdddf7C710a1eb` seems to be the right address.
![2021-05-17_23-31_1](https://user-images.githubusercontent.com/1658405/118559327-0d25e080-b768-11eb-8a8f-d68d439912c6.png)
![2021-05-17_23-31](https://user-images.githubusercontent.com/1658405/118559334-0f883a80-b768-11eb-80a3-34c05ddac7e6.png)

https://www.coingecko.com/en/coins/easyfi